interface/patient_tracker/patient_tracker.php

```diff
@@ -143,29 +143,30 @@
     //  Include Bootstrap and DateTimePicker
   call_required_libraries(array("jquery-min-3-1-1","bootstrap","datepicker"));
 ?>
-    <link rel='stylesheet' href='<?php echo $css_header ?>' type='text/css'>
-
+<link rel='stylesheet' href='<?php echo $css_header ?>' type='text/css'>
+<link rel="stylesheet" href="../../assets/css/iziModalToast/iziToast.min.css" type="text/css">
+<script type="text/javascript" src="../../assets/js/iziModalToast/iziToast.min.js" type="text/javascript"></script>
```

Comment:
You can call this in the `called_required_libraries()`

Reply:
Good suggestion! I just made this change and pushed it.